javaparser-testing/src/test/java/com/github/javaparser/ast/NodeListTest.java

```diff
@@ -219,4 +219,11 @@ public void replace() {
         assertEquals("z", list.get(1).asString());
         assertEquals("c", list.get(2).asString());
     }
+
+    @Test
+    public void toStringTest() {
+        final NodeList<Name> list = nodeList(new Name("abc"), new Name("bcd"), new Name("cde"));
+
+        assertEquals("abcbcdcde", list.toString());
```

Comment:
This could be instead be "[abc, bcd, cde]" with the changes I suggested

Reply:
Well, I did exactly what you agreed to in the issue, but I'll go fix it again then ;-)